src/provider/element-templates/components/ElementTemplatesGroup.js

```diff
@@ -143,24 +136,24 @@ function SectionToggle({ open }) {
  * @param {function} props.unlinkTemplate
  * @param {function} props.updateTemplate
  */
-function TemplateGroupButtons({ element, getTemplateId, unlinkTemplate, updateTemplate }) {
+function TemplateGroupButtons({ element, getTemplateId }) {
   const elementTemplates = useService('elementTemplates');
 
   const templateState = getTemplateState(elementTemplates, element, getTemplateId);
 
   if (templateState.type === 'NO_TEMPLATE') {
     return <SelectEntryTemplate element={ element } />;
   } else if (templateState.type === 'KNOWN_TEMPLATE') {
-    return <AppliedTemplate element={ element } unlinkTemplate={ unlinkTemplate } />;
+    return <AppliedTemplate element={ element } />;
   } else if (templateState.type === 'UNKNOWN_TEMPLATE') {
-    return <UnknownTemplate element={ element } unlinkTemplate={ unlinkTemplate } />;
+    return <UnknownTemplate element={ element } />;
   } else if (templateState.type === 'OUTDATED_TEMPLATE') {
     return (
       <OutdatedTemplate
         element={ element }
         templateState={ templateState }
-        unlinkTemplate={ unlinkTemplate }
-        updateTemplate={ updateTemplate } />
+        unlinkTemplate={ elementTemplates.unlinkTemplate }
```

Comment:
Any reason we don't use the same pattern like with `UnknownTemplate` and `AppliedTemplate` here?